agent/core/src/test/java/org/apache/shardingsphere/agent/core/plugin/PluginLifecycleServiceManagerTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.agent.core.plugin;
+
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLStreamHandlerFactory;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.jar.JarFile;
+import javax.management.loading.PrivateMLet;
+
+import net.bytebuddy.dynamic.loading.MultipleParentClassLoader;
+import org.apache.shardingsphere.agent.api.PluginConfiguration;
+import org.junit.Test;
+
+public final class PluginLifecycleServiceManagerTest {
+
+    @Test
+    public void assertInitPluginLifecycleService() {
+        Map<String, PluginConfiguration> pluginConfigs = new HashMap<>();
+        ArrayList<JarFile> pluginJars = new ArrayList<>();
```

Comment:
`ArrayList<JarFile>` should be declared as `Collection<JarFile>`

Reply:
In ShardingSphere, we generally do not use `ArrayList` unless we need to query data in the form of an array.
Can `new LinkedList` be used here to instead of `new ArrayList`?

Reply:
Changes done :)